### ISSUE TITLE
Week pick phrase

### ISSUE THREAD

**Issue by jbholden** [blreams/collegefootballpick10#75]
When players have submitted their spreadsheet picks in the past, they have also been able to submit a phrase like "GoJackets" with their picks.  Is this a feature we would like to add to the online pool?

We could add something to the Pick model that would only be valid for game number 10.  Then update the enter picks page code to allow a phrase to be entered.  (Any ideas on what to call the field?  phrase? )

I'm not exactly sure it would be displayed.  On a player's results page?  On the week results page in a another column on the righthand side?

**Comment by blreams** [comment]
How about "brag" for the name of the field. I'd rather put it in the Week model though, than in the Game model. If you add it to the Game model, then all 10 games for the week have the field but it is empty. It makes more sense to put in the Week model. That way it would Week.brag.

As for displaying it...the obvious choice is to put it on the right of the weekly points for the week results page. No need to display it any other time (except the edit picks page, of course).

**Comment by jbholden** [comment]
I don't think the Week model would work.  There is only 1 week model per week.  The requirement is for their to be a brag field for each player, so to me it makes sense to have it in the Pick model because there is a pick model for each player.   The field empty would be empty for picks for the game numbers 1-9.

**Comment by blreams** [comment]
Agreed, I didn't think through that clearly. The Pick model is the only place it could go.
